recipes/nng/all/conanfile.py

```diff
@@ -81,10 +87,14 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
+    def mbedtls(self):
+        return ["mbedtls::mbedtls"] if self.options.tls else []
+
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "nng"
         self.cpp_info.names["cmake_find_package_multi"] = "nng"
         self.cpp_info.libs = ["nng"]
+        self.cpp_info.requires = self.mbedtls()
```

Comment:
You don't need to do this when you're not using modules.
All requirements are automatically added.

Reply:
thanks @madebr !